src/daemons/MetaDaemon.cpp

```diff
@@ -117,10 +117,15 @@ int main(int argc, char *argv[]) {
     }
 
     auto *kvstore_ = kvstore.get();
+    auto *nebulaStore = dynamic_cast<nebula::kvstore::NebulaStore*>(kvstore_);
+    if (nebulaStore == nullptr) {
```

Comment:
do we not support cluster id when we not use nebula store?

Reply:
Now, cluster id store in  nebulaStore. Maybe, we can add clusterId support to other store.

Reply:
ClusterManager just dependence on the kvStore's get and put interface, so I think here no need cast.

Reply:
ClusterManager depends on Nebula::asyncMultiPut(KVStore has not this function), so dynamic_cast is needed.